retriever/lib/templates.py

```diff
@@ -112,6 +114,10 @@ def reference_url(self):
             if len(self.urls) == 1:
                 return '/'.join(self.urls[list(self.urls.keys())[0]].split('/')[0:-1]) + '/'
 
+    def printMessage(self):
+        if self.message!="":
```

Comment:
`""` evaluates to `False` in Python, so the Pythonic form of this conditional is `if self.message:`